static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -123,8 +128,17 @@ define([
                 }
             };
         },
+        buildUserSubscriptions = function (response) {
+            if (response && response.status !== 'error' && response.result && response.result.subscriptions) {
+                userListSubscriptions = pluck(response.result.subscriptions, 'listId');
```

Comment:
might be worth using `map` here, syntax would be the same (using a string instead of a fn gets that property), it's more well-known, and `pluck` has been dropped from lodash 4.